app/components/container/containerController.js

```diff
@@ -116,9 +116,23 @@ function ($scope, $state, $stateParams, $filter, Container, ContainerCommit, Ima
     });
   };
 
-  $scope.remove = function () {
+  $scope.confirmRemove = function () {
+    if ($scope.container.State.Running) {
+      ModalService.confirmDeletion(
+        'You are about to delete a running container.',
```

Comment:
I think we should keep the term 'Remove' as it's the term used by Docker. Just update the button label in ModalService to `Remove` instead of update.